File: src/ru/henridellal/dialer/ContactsEntryCache.java
package ru.henridellal.dialer;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

public class ContactsEntryCache {
	public final ImageView contactImage;
	public final TextView contactName;
	public final TextView phoneNumber;

	public ContactsEntryCache(View view) {
		contactImage = (ImageView) view.findViewById(R.id.contact_entry_image);
		contactName = (TextView) view.findViewById(R.id.contact_entry_name);
		phoneNumber = (TextView) view.findViewById(R.id.contact_phone_number);
	}
}

File: app/src/main/java/ru/henridellal/dialer/ContactsEntryCache.kt
package ru.henridellal.dialer

import android.view.View
import android.widget.ImageView
import android.widget.TextView

class ContactsEntryCache(view: View) {
    @JvmField
	val contactImage: ImageView = view.findViewById<View>(R.id.contact_entry_image) as ImageView

    @JvmField
	val contactName: TextView = view.findViewById<View>(R.id.contact_entry_name) as TextView

    @JvmField
	val phoneNumber: TextView = view.findViewById<View>(R.id.contact_phone_number) as TextView

}